conans/client/installer.py

```diff
@@ -356,8 +356,9 @@ def _raise_missing(self, missing):
 
         raise ConanException(textwrap.dedent('''\
             Missing prebuilt package for '%s'
-            Try to build from sources with "%s"
-            Or read "http://docs.conan.io/en/latest/faq/troubleshooting.html#error-missing-prebuilt-package"
+            Try to build from sources with '%s'
+            Use 'conan search <reference> --table table.html' and filter using your profile
```

Comment:
And something like _"Use 'conan search ....' to check the packages that are available"_... If it works, and you filter using your profile, you will get no results 🤷